### PR TITLE
Gui

### PR DESCRIPTION
**Pull request by Lauren34**

### CODE REVIEW

File: src/main/java/lesser/gameoflife/GameOfLifeComponent.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
+package lesser.gameoflife;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.*;
+
+public class GameOfLifeComponent extends JComponent {
+
+    private final GameOfLife game;
+    private final int cellSize = 20; // Size for each cell
```
Comment (andrewoid) [inline src/main/java/lesser/gameoflife/GameOfLifeComponent.java]: static

File: src/main/java/lesser/gameoflife/GameOfLifeComponent.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                    g.fillRect(x, y, cellSize, cellSize);
+                } else {
+                    g.drawRect(x, y, cellSize, cellSize);
+                }
+            }
+        }
+    }
+
+    // Start or stop the simulation
+    private boolean running = false;
```
Comment (andrewoid) [inline src/main/java/lesser/gameoflife/GameOfLifeComponent.java]: put this with the other member variables

File: src/main/java/lesser/gameoflife/GameOfLifeComponent.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+        for (int i = 0; i < grid.length; i++) {
+            for (int j = 0; j < grid[i].length; j++) {
+                int x = j * cellSize + xOffset;
+                int y = i * cellSize + yOffset;
+
+                if (grid[i][j] == 1) {
+                    g.fillRect(x, y, cellSize, cellSize);
+                } else {
+                    g.drawRect(x, y, cellSize, cellSize);
```
Comment (andrewoid) [inline src/main/java/lesser/gameoflife/GameOfLifeComponent.java]: try drawing grid lines, instead of outlining each empty square

**Review by andrewoid — COMMENTED** [review]
Add a readme and screenshot

**Comment by Lauren34** [comment]
> Add a readme and screenshot

I already added the file to my code and I see the README on git. Do you still not see it?

File: README.MD [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Game of Life
+![Screenshot 2024-09-16 at 6.11.27 PM.png](..%2F..%2F..%2F..%2Fvar%2Ffolders%2Fvd%2Fx5ywr8h943s87khlsgv197m00000gn%2FT%2FTemporaryItems%2FNSIRD_screencaptureui_TNJD1k%2FScreenshot%202024-09-16%20at%206.11.27%20PM.png)
+![Screenshot 2024-09-16 at 6.12.11 PM.png](..%2F..%2F..%2F..%2Fvar%2Ffolders%2Fvd%2Fx5ywr8h943s87khlsgv197m00000gn%2FT%2FTemporaryItems%2FNSIRD_screencaptureui_rXGcLK%2FScreenshot%202024-09-16%20at%206.12.11%20PM.png)
```
Comment (andrewoid) [inline README.MD]: This is not going to work when you upload it to GitHub

You also need to add your screenshots to git.

**Comment by Lauren34** [comment]
Ok, just fixed it
https://github.com/Lauren34/lesser-Game_Of_Life-2024/pull/2/commits/ffadb9a6c0c758e501be50e54ef41c3ce074e375

**Review by andrewoid — APPROVED** [review]

**Pull request merged**